README.md

```diff
@@ -12,6 +12,12 @@
 
 ### Save in ArchivesSpace
 
+The RAC_ASpace library improves upon the ArchivesSnake saving functionality by gerneralizing processes and making it easier to save new JSON data and push that back up to ArchivesSpace.
+
+We plan to do that by:
+  * Generalizing the transformation from a JSONModel object fetched by ASNake to editable JSON
+  * Simplify the process of replacing JSON content through the use of automated methods
```

Comment:
I'm curious about this. What kinds of things do you have in mind? I think this is where there might potentially be overlap with what we're calling data helpers.

Reply:
Yeah, I think there's some potential overlap here. I think what @daremiyo and I were thinking is that there's potential to move from abstraction layer to client layer and make updates in fewer steps that we're currently doing. I think there's potential to clean up the language here for clarity.